com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl

```diff
@@ -29,6 +29,36 @@ struct InputData
     half3   bakedGI;
     float2  normalizedScreenSpaceUV;
     half4   shadowMask;
+
+    #if defined(LIGHTMAP_ON)
+    half2   lightmapUV;
+    #else
+    float3  vertexSH;
+    #endif
+    #if defined(_NORMALMAP)
+    half3x3 tangentMatrixWS;
```

Comment:
We tend not to use matrix in shader naming. I suggest to more commong name tangentToWorld

Reply:
Fixed